gcm/build.gradle

```diff
@@ -41,7 +41,8 @@ android {
 }
 
 dependencies {
-    api "com.google.android.gms:play-services-gcm:$googleLibVersionGCM"
+    // last version for GCM to be supported
+    api "com.google.android.gms:play-services-gcm:12.0.1"
```

Comment:
it sounds like 15.0.0 brought in some 16.0 dependencies... hopefully this problem will be fixed soon

Reply:
That is strange. I sure hope so